CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- Dependency - Upgrade the `forest-express` dependency for intercom integration conversations retrieval.
```

Comment:
It is a fix for the end user, the dependency upgrade is a technical information.
We can understand this reading the commit, no need to mention it IMO. What is important is the intent/impact.
```suggestion
### Fixed
- Integrations - Fix intercom conversations display with API v2.
```
(same change description as forest-express in the end)